### PR TITLE
Add August 21, 2018

### PR DESCRIPTION
**Pull request by blackboxsw**

### CODE REVIEW

File: doc/2018/2018-08-21.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+
+## Spotlight:
```
Comment (powersj) [inline doc/2018/2018-08-21.md]: Can we say "Spotlight: Welcome Karl!"

File: doc/2018/2018-08-21.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+The week the ubuntu-server team welcomed a new member
+[Karl Stenerud](https://github.com/kstenerud) into the fold. He comes to us
+with around 20 years as a lead software developer. His broad experience with
+open source and enterprise software will be a great asset to the team, helping
+us drive distribution fixes into Ubuntu with greater velocity and quality.
+Per his github profile, let us all give our newest Ubuntu pirate a hearty
```
Comment (powersj) [inline doc/2018/2018-08-21.md]: GitHub

File: doc/2018/2018-08-21.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- netplan: Correctly render macaddress on a bonds and bridges when
+  provided. (LP: #1784699)
+- tools: Add 'net-convert' subcommand command to 'cloud-init devel'.
+- redhat: remove ssh keys on new instance. (LP: #1781094)
+- Use typeset or local in profile.d scripts. (LP: #1784713)
+- OpenNebula: Fix null gateway6 [Akihiko Ota] (LP: #1768547)
+
+## curtin
+
+- Don't allow reads of /proc and modprobe zfs through
```
Comment (raharper) [inline doc/2018/2018-08-21.md]: unittest: Don't allow ....

File: doc/2018/2018-08-21.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+
+## Spotlight:
+The week the ubuntu-server team welcomed a new member
```
Comment (dpb1) [inline doc/2018/2018-08-21.md]: canonical-server

File: doc/2018/2018-08-21.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+[Karl Stenerud](https://github.com/kstenerud) into the fold. He comes to us
```
Comment (dpb1) [inline doc/2018/2018-08-21.md]: add parenthetical somewhere... (kstenerud on freenode)

File: doc/2018/2018-08-21.md (hunk 4 above)
Comment (blackboxsw) [reply doc/2018/2018-08-21.md]: done

File: doc/2018/2018-08-21.md (hunk 5 above)
Comment (blackboxsw) [reply doc/2018/2018-08-21.md]: good point

File: doc/2018/2018-08-21.md (hunk 1 above)
Comment (blackboxsw) [reply doc/2018/2018-08-21.md]: done

**Review by blackboxsw — COMMENTED** [review]
addressed comments thank you

**Pull request merged**